### PR TITLE
Adds Kibana pull and issue attributes

### PR DESCRIPTION
**Pull request by KOTungseth**
Added the following attributes:
:kibana-issue: {kib-issue}
:kibana-pull:  {kib-pull}

### CODE REVIEW

**Comment by gtback** [comment]
Do we need both `kib-pull` and `kibana-pull`?

File: shared/attributes.asciidoc [hunk 1]
```diff
@@ -336,18 +336,19 @@ Reusable images
...
-:ml-pull:    {ml-repo}pull/
-:ml-commit:  {ml-repo}commit/
+:es-repo:     https://github.com/elastic/elasticsearch/
+:es-issue:    {es-repo}issues/
+:es-pull:     {es-repo}pull/
+:es-commit:   {es-repo}commit/
+:kib-repo:    https://github.com/elastic/kibana/
+:kib-issue:   {kib-repo}issues/
+:kib-pull:    {kib-repo}pull/
+:kibana-pull: {kib-repo}pull/
```
Comment (lcawl) [inline shared/attributes.asciidoc:346]: Not mandatory, but you could make the dependency between these attributes explicit by doing this:
```suggestion
:kibana-pull: {kib-pull}
```

**Comment by lcawl** [comment]
> Do we need both `kib-pull` and `kibana-pull`?

If not, we must go back and fix any usage of "kib-pull".  For example, I see it in the 7.8.0 release notes.

**Comment by lcawl** [comment]
https://github.com/elastic/kibana/pull/74306 builds successfully with these changes

**Pull request merged**